guide/installation-and-configuration/pre-installation-checklist.markdown

```diff
@@ -8,7 +8,9 @@ tags: [guide, installation]
 
 ## Download Packages
 
-[Download CFEngine](http://cfengine.com/product/free-download)
+[Download CFEngine][d] packages and [verify their signatures][Verifying package signatures].
+
+[d]: http://cfengine.com/product/free-download
```

Comment:
I thought you dropped this inline reference, We already have a reference target for download, you used it below:

`On the [Download CFEngine][enterprise software download page], you will find`